semantics-3.0/Semantics4.hs

```diff
@@ -1,11 +1,12 @@
 {-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
These extensions will probably not translate to Isabelle and I do not know how they will translate to Purescript.

Reply:
Fair enough, I'll remove it.